tools/mason/MasonUtils.chpl

```diff
@@ -86,7 +86,7 @@ proc makeTargetFiles(binLoc: string, projectHome: string) {
 
 proc stripExt(toStrip: string, ext: string) : string {
   if toStrip.endsWith(ext) {
-    var stripped = toStrip[..toStrip.size - ext.size];
+    var stripped = toStrip[..toStrip.size - ext.size - 1];
```

Comment:
Why not use `<`?
```suggestion
    var stripped = toStrip[..<(toStrip.size - ext.size)];
```

Reply:
Because `..<` didn't exist 4 months ago when I made this commit and I didn't take the time this month to go back and update all 1000+ files that were sensitive to 0- vs 1-based indexing changes to review them for optimal style.

Reply:
Updated to the new syntax in https://github.com/chapel-lang/chapel/pull/14671/commits/ca4cc8b03444de04c412fd650546434d4d753353